docs/quickstarts/insights-malware-install/insights-malware-install.yml

```diff
@@ -0,0 +1,206 @@
+# Additional info: https://docs.openshift.com/container-platform/4.9/web_console/creating-quick-start-tutorials.html
+metadata:
+  name: insights-malware-install
+  # you can add additional metadata here
+  instructional: true
+spec:
+  displayName: Install the Malware Service
+  durationMinutes: 5
+  # Optional type section, will display as a tile on the card
+  type:
+    text: Quick start
+    # 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'grey'
+    color: grey
+  icon: data:image/svg+xml;base64,PCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjUxMiIgaGVpZ2h0PSI1MTIiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj4KPHRpdGxlPjwvdGl0bGU+CjxnIGlkPSJpY29tb29uLWlnbm9yZSI+CjwvZz4KPHBhdGggZD0iTTQ0OCA2NHY0MTZoLTMzNmMtMjYuNTEzIDAtNDgtMjEuNDktNDgtNDhzMjEuNDg3LTQ4IDQ4LTQ4aDMwNHYtMzg0aC0zMjBjLTM1LjE5OSAwLTY0IDI4LjgtNjQgNjR2Mzg0YzAgMzUuMiAyOC44MDEgNjQgNjQgNjRoMzg0di00NDhoLTMyeiI+PC9wYXRoPgo8cGF0aCBkPSJNMTEyLjAyOCA0MTZ2MGMtMC4wMDkgMC4wMDEtMC4wMTkgMC0wLjAyOCAwLTguODM2IDAtMTYgNy4xNjMtMTYgMTZzNy4xNjQgMTYgMTYgMTZjMC4wMDkgMCAwLjAxOS0wLjAwMSAwLjAyOC0wLjAwMXYwLjAwMWgzMDMuOTQ1di0zMmgtMzAzLjk0NXoiPjwvcGF0aD4KPC9zdmc+Cg==
+  prerequisites:
+    - You must have sudo access to the system.
+    - You must be a member of a User Access group with the Malware detection administrator role.
+  description: |-
+    Install YARA and the malware detection controller, test, scan and view your results.
+  introduction: |-
+    **Welcome to the quick start that will enable scanning for Malware signatures. 
+    In this quick start, you will configure user access, install packages, and ensure that the Insights client and systems are configured correctly. 
+  
+  tasks:
+    - title: Configure User Access
+      description: |- 
+
+        An organization administrator or User Access administrator must add users to a User Access group with the required roles. 
+        The administrator must also have sudo access to the system. 
+
+        **To create a custom role:**
+
+        1. Navigate to the **Red Hat Hybrid Cloud Console**.
```

Comment:
To access the quick start, you'll already be in the Hybrid Cloud Console - this step can be deleted.